### DISCUSSION TITLE
Resetting password?

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#440] [Q&A] [answered]
One of my colleagues is trying to reset her password. She followed the link in the e-mail she got en ran in to the following issue:
_The recovery token is invalid or has already been used. Please retry the flow._

She tried a second time but the same notification appeared.

![image002](https://user-images.githubusercontent.com/87815729/136003123-50dbe2e8-6ac6-48b6-a7c3-6d45013efe67.jpg)

**Comment by Comoque1** [comment]
Hi Flori, just double checked with my account and was able to successfully recover the password.
This error is thrown, when the link is clicked more than once. I think, the other reason could be, that the the email with the link was clicked 15 minutes after it was sent.
Would it be possible to request new [password reset](https://hub.alkem.io/identity/recovery), and make sure that she is following the latest email link?

**Comment by techsmyth** [comment, marked as answer]
Just adding to Evgeni's reply: indeed it looks like she needs to re-submit her password reset request. For clarity the link in the text above is https://hub.alkem.io/identity/recovery. 

If the issue still persists please ask her to send an email to support@alkem.io  (which Evgeni will get) and include there her email address so we can dive deeper.

**Reply by techsmyth** [reply]
@FloriSpoelstra is there any update on this user being able to login?

**Reply by FloriSpoelstra** [reply]
Yes, we resolved this one with the help of Evgeni. The recovery link kept failing so we removed the account and created it again. 

We haven't worked out why it did not work. I sat next to my colleague to walk her through it but we kept getting a "the recovery token is invalid" respons. I haven't been able to recreate the problem, when I reset my own password it does work.
